k8s/elastic/Chart.yaml

```diff
@@ -20,3 +20,8 @@ dependencies:
     version: 0.20.1
     condition: vector.enabled
     repository: https://helm.vector.dev
+  - name: metricbeat
+    version: 8.5.1
```

Comment:
change version to `7.17.3` to be consistent with the other parts of Elastic stack. Not sure we are ready to move to 8.x elastic

Reply:
fixed